### PR TITLE
Improved ArcGIS feature for support servers older than version 11

### PR DESCRIPTION
**Pull request by mdastous-bentley**
Several small bug fixes:

 **Server v10.91 issue:**
- PBF requests makes the server return an error, use JSON request instead.
- Server requires all request parameters to be correctly URI encoded (Not an issue with Server v11)
- GetFeatureInfo requests require unit of length to be explicitly specified.

**Other issues:**
- Some servers report a maxLOD of 0, which should interpreted as 'none'.
- Issue applying default symbology when the provided one is not supported.
- Changed the default 'Point marker sybol' to a blue pushpin to make it more visible.

### CODE REVIEW

**Comment by mdastous-bentley** [comment]
@calebmshafer I really wish this get included in 3.5,

**Review by pmconne — APPROVED** [review]

File: extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts [hunk 1]
```diff
@@ -139,26 +139,52 @@ export class ArcGisFeatureProvider extends ArcGISImageryProvider {
       }
     }
 
+    // Parse server version
+    let majorVersion: number|undefined;
+    if (this.serviceJson?.currentVersion) {
+      try {
+        majorVersion = Math.trunc(this.serviceJson?.currentVersion);
```
Comment (aruniverse) [inline extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts:146]: does this actually throw? guess it doesnt hurt to wrap incase of malformed serviceJson but itd have to be pretty screwed up

**Review by aruniverse — APPROVED** [review]

File: extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts [hunk 2]
```diff
@@ -167,8 +193,9 @@ export class ArcGisFeatureProvider extends ArcGISImageryProvider {
     const scales = ArcGisUtilities.getZoomLevelsScales(this.defaultMaximumZoomLevel, this.tileSize, minScale, maxScale);
     if (scales.minLod)
       this._minDepthFromLod = scales.minLod;
-    if (scales.maxLod)
-      this._maxDepthFromLod = scales.maxLod;
+
+    // Some servers advertises a max LOD of 0, it should be interpreted as 'not defined' (otherwise a max lod of 0 would would mean never display anything)
+    this._maxDepthFromLod =  (scales.maxLod ? scales.maxLod : this.defaultMaximumZoomLevel);
```
[the remark this answers is not in the document]
Comment (pmconne) [reply extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts:198]: @aruniverse see comment about max LOD of 0.

[the remark this answers is not in the document]
Comment (aruniverse) [reply extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts:198]: updated suggestion; @mdastous-bentley  feel free to ignore as well as other comment

File: extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts (hunk 1 above)
Comment (mdastous-bentley) [reply extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts:146]: yea I dont know, just wanted to avoid any mess.

**Pull request merged**